target/scripts/helpers/accounts.sh

```diff
@@ -157,7 +157,7 @@ function _create_dovecot_alias_dummy_accounts
       fi
 
       DOVECOT_USERDB_LINE="${ALIAS}:${REAL_ACC[1]}:5000:5000::/var/mail/${REAL_DOMAINNAME}/${REAL_USERNAME}::${REAL_ACC[2]:-}"
-      if grep -qF "${DOVECOT_USERDB_LINE}" "${DOVECOT_USERDB_FILE}"
```

Comment:
Why was the `-F` option removed? IIRC @casperklein mentioned (please correct me if I‘m wrong), that we need to search for the pattern literally, i.e. with fixed strings. If this is indeed not needed, all is well :D

Reply:
`-F` is useful/best practice, when no regex is used and just plain text should be matched. So when adding `^`, `-F` must be removed 👍

Reply:
Ah yes, right. Would it make sense then 
to add a `$` at the end here too?

Reply:
If you want to match the whole line exactly, `grep -qxF` would work (same as if we additionally had `^`+`$` tacked on both ends in a compatible manner.

However this specific check is only interested in the duplicate user entry really, so we don't need to match the entire line.. just the first column. A key lookup such as `grep -qi "^${ALIAS}:" "${DOVECOT_USERDB_FILE}"` would suffice.